### DISCUSSION TITLE
FetchError: Unexpected token o in JSON at position 1

### DISCUSSION THREAD

**Discussion by riancz** [supabase/supabase#4609] [Questions]
Hi guys lately I am getting this error:
`FetchError: Unexpected token o in JSON at position 1`

The weird part is that I probably didn't do anything wrong because this error appears randomly for several hours and then disappears again.

Now I am not able to fetch data, log in or do pretty much anything. My DB is functioning though. When I go to app.supabase.io everything seems fine, albeit little laggy. Anyone else have these problems?

<img width="379" alt="image" src="https://user-images.githubusercontent.com/47085300/146922550-31e99157-e326-416c-b670-bf6cf3775b59.png">

**Comment by jorgebaraltq** [comment]
I have the same problem, how did you solve it? @riancz

**Reply by steve-chavez** [reply]
I recall this can happen when the REST API is overloaded and then Cloudflare sends an HTML when the request times out. If that's the case then solving it would be a matter of upgrading to a higher tier(for more resources) or optimizing your queries(so the REST API responds faster).
